tests/Doctrine/Migrations/Tests/Generator/SqlGeneratorTest.php

```diff
@@ -23,48 +23,77 @@
     /** @var SqlGenerator */
     private $migrationSqlGenerator;
 
-    public function testGenerate() : void
-    {
-        $sql = [
-            'SELECT 1',
-            'SELECT 2',
-            'UPDATE table SET value = 1 WHERE name = 2 and value = 1 and field = 2 and active = 1',
-            'SELECT * FROM migrations_table_name',
-        ];
+    /** @var array */
+    private $sql;
 
-        $formattedUpdate = SqlFormatter::format($sql[2], false);
+    public function testGenerate(): void
+    {
 
-        $expectedCode = <<<'CODE'
+        $expectedCode = $this->prepareGeneratedCode(
+            <<<'CODE'
 $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
 
 $this->addSql('SELECT 1');
 $this->addSql('SELECT 2');
 $this->addSql('%s');
-CODE;
-
-        $expectedCode = sprintf($expectedCode, $formattedUpdate);
-
-        $this->configuration->expects(self::any())
-            ->method('getMigrationsTableName')
-            ->willReturn('migrations_table_name');
+CODE
+        );
 
         $this->platform->expects(self::once())
             ->method('getName')
-            ->willReturn('mysql');
+            ->willReturn('mysql')
+        ;
 
-        $code = $this->migrationSqlGenerator->generate($sql, true, 80);
+
+        $code = $this->migrationSqlGenerator->generate($this->sql, true, 80);
 
         self::assertSame($expectedCode, $code);
     }
 
-    protected function setUp() : void
+    public function testGenerationWithoutCheckingDatabasePlatform(): void
+    {
+        $expectedCode = $this->prepareGeneratedCode(
+            <<<'CODE'
+$this->addSql('SELECT 1');
+$this->addSql('SELECT 2');
+$this->addSql('%s');
+CODE
+        );
+
+        $code = $this->migrationSqlGenerator->generate($this->sql, true, 80, false);
+
+        self::assertSame($expectedCode, $code);
+    }
+
+    protected function setUp(): void
     {
         $this->configuration = $this->createMock(Configuration::class);
-        $this->platform      = $this->createMock(AbstractPlatform::class);
+        $this->platform = $this->createMock(AbstractPlatform::class);
```

Comment:
Unrelated change. I'll also assume that this is going to make the Coding Standards check fail, if travis-ci ever decides to pick up on this PR...